### DISCUSSION TITLE
API Endpoint Problema campo number all'interno di [List Issued Documents, get /c/{company_id}/issued_documents]

### DISCUSSION THREAD

**Discussion by Kankro** [fattureincloud/api#380] [Q&A] [answered]
Salve a tutti,
ho notato che per qualche motivo, quando testo questo endpoint con dei filtri di configurazione `/c/{company_id}/issued_documents `, non mi ritorna il numero della proforma.
url https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents
Se compilo i field in questo modo: 
```
type: proforma
fieldset: id,year,number,amount_net,amount_vat,amount_gross,date,next_due_date,created_at,updated_at
sort: -updated_at
page: 1
per_page: 100
```

Mi ritorna
```
{
  "current_page": 1,
  "data": [
    {
      "id": 1234,
      "year": 2024,
      ....
      "number": null,
      ....
    }
  ],
  ....
}
```

Ma se invece setto  `fieldset: detailed`
Mi ritorna
```
{
  "current_page": 1,
  "data": [
    {
      "id": 1234,
      "type": "proforma",
      "year": 2024,
      ....
      "number": 2
      ....
    }
  ],
  ....
}
```

Se setto invece di proforma a fattura funziona senza problemi.

**Comment by emajo** [comment, marked as answer]
Ciao @Kankro le chiederei di aggiungere anche il campo type ai campi richiesti, in questo modo dovrebbe funzionare.

Grazie,
Emanuele.

**Reply by Kankro** [reply]
Buondì @emajo , il campo type è obbligatorio e l'ho settato a `proforma`, solo nel caso in cui il fieldset sia settato a `detailed` mi ritorna l'identificativo della proforma.

![Screenshot 2023-12-05 at 9 58 09 AM](https://github.com/fattureincloud/api/assets/26323857/d5859f02-b13d-4cda-aa45-87613b39292e)

**Reply by Kankro** [reply]
Ok, scusa ho capito ora consa intendevi così funziona 👍
Grazie.
